multi/imxrt-multi/config.h

```diff
@@ -26,186 +28,668 @@
 #include "imxrt1060.h"
 #endif
 
+#include <board_config.h>
+
+
 #ifndef IMXRT_MULTI_PRIO
 #define IMXRT_MULTI_PRIO 2
 #endif
 
 /* UART */
 
 #ifndef UART1
-#define UART1 1
+#define UART1          0
+#define UART1_BAUDRATE 0
+#elif !ISBOOLEAN(UART1)
+#error "UART1 must have a value of 0, 1, or be undefined"
+#endif
+
+#if UART1
+
+#ifndef UART1_TX_PIN
+#warning "[Deprecated] UART1_TX_PIN should be explicitly defined in board_config.h. Using the default"
+#define UART1_TX_PIN UART1_TX_PIN_DEFAULT
+#elif ISEMPTY(UART1_TX_PIN)
+#error "UART1_TX_PIN must not be empty"
+#endif
+
+#ifndef UART1_RX_PIN
```

Comment:
not sure what would be more readable ^C^V almost the same configuration 12 times or specifying it once in external file and including 12 times, eg:

```c
#define _UART_N 1
#include "_uart_def.h"
#undef _UART_N
```

Edit:
I think we could keep it as-is, a the code is simple (albeit kindof long) with comment describing what can be set in `board_config.h`. Probably we will migrate to device tree in the future, so let's not over-complicate the solution right now.